opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -311,3 +311,57 @@ def _convert_aggregation_temporality(
             aggregation_temporality=aggregation_temporality,
         )
     return None
+
+
+class _AggregationFactory(ABC):
+    @abstractmethod
+    def create_aggregation(self) -> Aggregation:
```

Comment:
```suggestion
    def create_aggregation(self) -> _Aggregation:
```
The parent class and the current aggregations should be made private since they are not exposed to the user.